modules/social_features/social_group/modules/social_group_request/src/Plugin/Block/SocialGroupRequestMembershipNotification.php

```diff
@@ -157,8 +160,15 @@ public function build() {
       '#tag' => 'div',
       '#value' => $this->t('There @link to join this group.', [
         '@link' => Link::fromTextAndUrl(
-          $this->translation->formatPlural($requests, 'is (1) new request', 'are (@count) new requests'),
-          Url::fromRoute('view.group_pending_members.membership_requests', ['arg_0' => $this->group->id()])
+          $this->translation->formatPlural(
```

Comment:
@OH17L When I created Flexible group with Request o join method on the test env of this PR i got an error:

`Call to a member function formatPlural() on null in Drupal\social_group_request\Plugin\Block\SocialGroupRequestMembershipNotification->build() (line 163 of profiles/contrib/social/modules/social_features/social_group/modules/social_group_request/src/Plugin/Block/SocialGroupRequestMembershipNotification.php).
`

Reply:
Fixed.